CONTRIBUTING.md

```diff
@@ -23,3 +23,29 @@ If you'd like to help make RSpec better, here are some ways you can contribute:
 If you need help getting started, check out the [DEVELOPMENT](DEVELOPMENT.md) file for steps that will get you up and running.
 
 Thanks for helping us make RSpec better!
+
+## `Small` issues
+
+These issues are the ones that we be believe are best suited for new
+contributors to get started on. They represent a potential meaningful
+contribution to the project that should not be too hard to pull off.
+
+## `Triage` issues
+
+These issues are ones that have been labelled by the maintainers that we
+believe do not currently have enough information to be reproduced the RSpec
+team. While not directly counted by the GitHub contribution graph, we consider
+helping us triage issues to the point where they can be reproduced as an
+extremely meaningful contribution.
+
+## `Has reproduction case` issues
+
+Issues that have reproduction cases have a repository that we can clone that
+enable us to quickly determine the issue is valid and then start debugging
+within RSpec. These issues are good ones to tackle to help us actively fix bugs.
+
+## Maintenance branches
```

Comment:
this seems very different than the previous sections, you might consider putting this is a different section. it is a not a issue label so perhaps "dev env" or something like this is a better move. either way i'd get it out of this section.